content/faq.md

```diff
@@ -2,25 +2,27 @@
 title = "Frequently Asked Questions (FAQ)"
 +++
 
+**This article has been verified for VCMI 1.1 (release date 23.12.2022)**
+
 * Again, what is VCMI?
 
-> In simple words: Remake of Heroes of Might and Magic 3. We use original Heroes 3 graphic, sound and text assets and make new game that is supposed to work 100% same as H3. Everything in VCMI is made from scratch - loading game files, drawing graphics, AI, game mechanics etc. We also plan supporting WoG mod features, but for now there are areas in VCMI that need more attention than WoG features.
+> In simple words: Remake of Heroes of Might and Magic 3. We use original Heroes 3 graphic, sound and text assets and make new game that is supposed to work 100% same as H3. Everything in VCMI is made from scratch - loading game files, drawing graphics, AI, game mechanics etc. We also plan supporting WoG, HD mod and other cool features, but there are other areas that need our attention as well.
 
 * What is the purpose of VCMI? Why to play or develop it if Heroes 3 is already there?
 
-> Our goal is to make "better" heroes 3 that overcomes various H3 engine limitations and allows way easier extensibility / modding than original game. Also VCMI works without problems on Windows, Linux, MacOS and Android.
+> Our goal is to make "better" heroes 3 that overcomes various H3 engine limitations and allows way easier extensibility / modding than original game. Also VCMI is multiplatform so it works on Windows, Linux, MacOS, iOS and Android.
 
 * Is HD Mod / HotA / WoG compatible with VCMI?
 
-> No, HD Mod, HotA and WoG are mods that work with original Heroes 3, VCMI is new game. Remaking these in VCMI would be possible, but there are bigger priorities. Note that some VCMI-specific modding features may also allow remaking some part of these mods by modders as side-effects. For example VCMI scripting while not being WoG focused work, will be replacement for WoG's scripting feature.
+> No, HD Mod, HotA and WoG are mods that work with original Heroes 3, VCMI is new game. Remaking these in VCMI would be possible, but there are bigger priorities. Note that some VCMI-specific modding features may also allow remaking some part of these mods by modders as side-effects. For example VCMI scripting (planned future feature) while not being WoG focused work, will be replacement for WoG's scripting feature.
```

Comment:
I'd split it into two qiestions:

- HotA and WoG are very different platforms. Still, they were ported by enthusiatsts as VCMI mods, offerting limited range of their features. You can't just install VCMI over WoG or HotA binaries.

- You can't install HD Mod over VCMI - VCMI provides its own options for high resolutions and quality-of-life GUI improvements.